src/robusta/integrations/kubernetes/base_triggers.py

```diff
@@ -96,7 +178,20 @@ def should_fire(self, event: TriggerEvent, playbook_id: str, build_context: Dict
         if not exact_match(self.operation, K8sOperationType(k8s_payload.operation)):
             return False
 
+        accept = True
```

Comment:
Since this check might be expensive, with path selector, I would put it last in the function (after all other checks)

Reply:
done